<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="OptionsWidgetGLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="binFileHLayout">
     <item>
      <widget class="QLabel" name="binFileLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Binary file:</string>
       </property>
       <property name="buddy">
        <cstring>binFileEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="binFileEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="binSelectButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="onBuildCheckBox">
     <property name="text">
      <string>Check project on build</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="onSaveCheckBox">
     <property name="text">
      <string>Check document on save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="onProjectChangeCheckBox">
     <property name="text">
      <string>Check on active project change</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="onFileAddedCheckBox">
     <property name="text">
      <string>Check added files</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="unusedCheckBox">
     <property name="text">
      <string>Check for unused functions</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="inconclusiveCheckBox">
     <property name="text">
      <string>Check for inconclusive errors</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="customParametersHLayout">
     <item>
      <widget class="QLabel" name="customParametersLabel">
       <property name="text">
        <string>Custom parameters:</string>
       </property>
       <property name="buddy">
        <cstring>customParametersEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customParametersEdit"/>
     </item>
     <item>
      <widget class="QToolButton" name="getHelpButton">
       <property name="text">
        <string>?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="ignoreEditHLayout">
     <item>
      <widget class="QLabel" name="ignoreEditLabel">
       <property name="text">
        <string>Ignore patterns:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ignoreEdit">
       <property name="toolTip">
        <string>Comma separated wildcards</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="showOutputCheckBox">
     <property name="text">
      <string>Show binary's output</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QCheckBox" name="popupOnErrorCheckBox">
     <property name="text">
      <string>Popup issues pane when errors found</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QCheckBox" name="popupOnWarningCheckBox">
     <property name="text">
      <string>Popup issues pane when warnings found</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>266</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="showIdCheckBox">
     <property name="text">
      <string>Show message Id on Issues</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>binFileEdit</tabstop>
  <tabstop>binSelectButton</tabstop>
  <tabstop>onBuildCheckBox</tabstop>
  <tabstop>onProjectChangeCheckBox</tabstop>
  <tabstop>onSaveCheckBox</tabstop>
  <tabstop>onFileAddedCheckBox</tabstop>
  <tabstop>unusedCheckBox</tabstop>
  <tabstop>inconclusiveCheckBox</tabstop>
  <tabstop>customParametersEdit</tabstop>
  <tabstop>getHelpButton</tabstop>
  <tabstop>ignoreEdit</tabstop>
  <tabstop>showOutputCheckBox</tabstop>
  <tabstop>showIdCheckBox</tabstop>
  <tabstop>popupOnErrorCheckBox</tabstop>
  <tabstop>popupOnWarningCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
